src/XMakeTasks/Microsoft.CSharp.CrossTargeting.targets

```diff
@@ -10,15 +10,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 ***********************************************************************************************
 -->
 
+<!-- These targets are deprecated, forward to new targets -->
```

Comment:
Is this intended as a temporary measure so we can update the SDK gradually? Since we haven't shipped an official release with these targets I hope it'd be ok to just rename them atomically.